lib/src/main/java/com/orange/ods/compose/component/button/OdsIconToggleButtonsRow.kt

```diff
@@ -95,45 +91,81 @@ fun OdsIconToggleButtonsRow(
-    displaySurface: OdsDisplaySurface,
-    onClick: (Int) -> Unit
-) {
-    val iconTint = buttonToggleIconColor(displaySurface, selected, iconToggleButton.enabled)
-    val backgroundAlpha = if (selected && iconToggleButton.enabled) 0.12f else 0f
-    val iconToggleButtonStateDescription = selectionStateDescription(selected = selected)
-
-    Icon(
-        modifier = Modifier
-            .background(
-                color = buttonToggleBackgroundColor(displaySurface)
-                    .copy(alpha = if (iconToggleButton.enabled) animateFloatAsState(backgroundAlpha, label = "").value else backgroundAlpha)
-            )
-            .padding(12.dp)
-            .run {
-                if (iconToggleButton.enabled) {
-                    clickable(interactionSource = remember { DisabledInteractionSource() }, indication = null) { onClick(index) }
-                } else {
-                    this
+    /**
+     * Creates an instance of [OdsIconToggleButtonsRowIcon].
+     *
+     * @param painter Painter of the icon.
+     * @param contentDescription The content description associated to this [OdsIconButtonIcon].
+     * @param enabled Whether or not this [OdsIconToggleButtonsRowIcon] will handle input events and appear enabled for
+     * semantics purposes, true by default.
+     */
+    constructor(painter: Painter, contentDescription: String, enabled: Boolean = true) : super(painter as Any, contentDescription, enabled)
+
+    /**
+     * Creates an instance of [OdsIconButtonIcon].
+     *
+     * @param imageVector Image vector of the icon.
+     * @param contentDescription The content description associated to this [OdsIconButtonIcon].
+     * @param enabled Whether or not this [OdsIconToggleButtonsRowIcon] will handle input events and appear enabled for
+     * semantics purposes, true by default.
+     */
+    constructor(imageVector: ImageVector, contentDescription: String, enabled: Boolean = true) : super(imageVector as Any, contentDescription, enabled)
+
+    /**
+     * Creates an instance of [OdsIconButtonIcon].
+     *
+     * @param bitmap Image bitmap of the icon.
+     * @param contentDescription The content description associated to this [OdsIconButtonIcon].
+     * @param enabled Whether or not this [OdsIconToggleButtonsRowIcon] will handle input events and appear enabled for
+     * semantics purposes, true by default.
+     */
+    constructor(bitmap: ImageBitmap, contentDescription: String, enabled: Boolean = true) : super(bitmap as Any, contentDescription, enabled)
+
+    private var index: Int = 0
+    private var onClick: (Int) -> Unit = {}
+    private var selected: Boolean = false
+
+    override val tint: Color
+        @Composable
+        get() {
+            return buttonToggleIconColor(displaySurface, selected, enabled)
+        }
+
+    @Composable
+    internal fun Content(index: Int, displaySurface: OdsDisplaySurface = OdsDisplaySurface.Default, selected: Boolean = false, onClick: (Int) -> Unit) {
```

Comment:
If we strictly follow the guidelines, `onClick` should not be the last parameter but it's nice to have a trailing closure nevertheless. Thus I don't know which rule should be applied here.

Reply:
As `OnClick` is a mandatory parameter, I think we have to follow the first rule. I change this one.